db.sql

```diff
@@ -49,15 +49,16 @@ CREATE TABLE "Survey" (
 );
 
 CREATE VIEW attendees_view AS (
-	SELECT
-		p. "userId" AS presenter,
-		p. "id" AS preso,
-		a. "id" AS attendee,
-		a. "email" AS email,
-		a. "fullName" AS "name",
-		a."createdAt" As created_at
-	FROM
-		"Preso" p
-		JOIN "Survey" s ON p.id = s. "presoId"
-		JOIN "Attendee" a ON s. "attendeeId" = a.id
+    SELECT p."userId" AS presenter,
```

Comment:
PR #62 means we will no longer need to track this file. We'll favor using the Supabase CLI to manage migrations, so this file will be removed.